.env.example

```diff
@@ -23,6 +23,10 @@ REACT_APP_ENVIRONMENT=mainnet
 #VHS endpoint url
 REACT_APP_DATA_URL=https://data.xrpl.org/v1/network
 
+# Whether to use ws instead of wss
+# Only used locally (the deployed Explorer requires wss)
+REACT_APP_INSECURE_WS=0
```

Comment:
```suggestion
# Whether to use ws instead of wss
# Only used locally (the deployed Explorer requires wss)
# 1 = ws, 0 = wss (default)
REACT_APP_USE_INSECURE_WS=0
```

Two suggestions here:
1. Clarifying that we're using 0 as a boolean with the examples of how to use it
2. Adding a word to the name to clarify that this is not a string variable like the other `REACT_APP_...` and is instead a boolean